.github/workflows/auto-merge.yml

```diff
@@ -14,9 +14,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Git checkout
```

Comment:
why do we need to checkout the code?
this action uses github API to perform the merge automatically

Reply:
Curiosity at work :smile:. You're right, this was not the right direction.